src/transferslot.cpp

```diff
@@ -216,13 +216,17 @@ TransferSlot::~TransferSlot()
                         break;
 
                     case REQ_DECRYPTING:
+                    {
                         LOG_info << "Waiting for block decryption";
                         std::mutex finalizedMutex; 
                         std::unique_lock<std::mutex> guard(finalizedMutex);
                         auto outputPiece = transferbuf.getAsyncOutputBufferPointer(i);
                         outputPiece->finalizedCV.wait(guard, [&](){ return outputPiece->finalized; });
                         downloadRequest->status = REQ_DECRYPTED;
                         break;
+                    }
+
+                    default: ;
```

Comment:
I'm aware the last case of a switch doesn't require an explicit `break`, but I think it's better for maintenance and to avoid surprises, in case someone add a new case below `default`, without notice the it has no break statement :)
```suggestion
                    default: 
                        break;
```

Reply:
A new case would normally go above the default, and getting `break` right in that case would be that dev's responsibility, but I'll make the change rather than spend time arguing ;-)